agency/README.md

```diff
@@ -76,20 +85,21 @@ If `device_id` is specified, `GET` will return an array with a single vehicle re
 ```
 
 A vehicle record is as follows:
-
-| Field         | Type      | Field Description                                                             |
-| ------------- | --------- | ----------------------------------------------------------------------------- |
-| `device_id`   | UUID      | Provided by Operator to uniquely identify a vehicle                           |
-| `provider_id` | UUID      | Issued by Agency and [tracked](../providers.csv)                              |
-| `vehicle_id`  | String    | Vehicle Identification Number (vehicle_id) visible on vehicle                 |
-| `vehicle_type`        | Enum      | [Vehicle Type][vehicle-types]                                                 |
-| `propulsion_types`  | Enum[]    | Array of [Propulsion Type][propulsion-types]; allows multiple values          |
-| `year`        | Integer   | Year Manufactured                                                             |
-| `mfgr`        | String    | Vehicle Manufacturer                                                          |
-| `model`       | String    | Vehicle Model                                                                 |
-| `state`       | Enum      | Current vehicle state. See [Vehicle State][vehicle-states]                    |
-| `prev_events`  | Enum[]      | Last [Vehicle Event][vehicle-event]                                           |
-| `updated`     | [timestamp][ts] | Date of last event update                                                     |
+| Field                   | Type            | Field Description                                                                                    |
+|-------------------------|-----------------|------------------------------------------------------------------------------------------------------|
+| `accessibility_options` | Enum[]          | Accessibility options available for this vehicle, see [Accessibility Options][accessibility-options] |
```

Comment:
I'm having a hard time wrapping my head around what `vehicle_attributes` and `trip_attributes` should even look like. Should it be like:
`[[key1, val1], [key2, val2]]`
or
`[{ key1: val 1}, { key2: val2 }]`

or something else? This feels like it should probably be an _object_ rather than an array, so:
`{ key1: val1, key2: val2 }`